core/config/envvar/schema.go

```diff
@@ -125,7 +127,7 @@ type ConfigSchema struct {
 	KeeperRegistrySyncUpkeepQueueSize          uint32          `env:"KEEPER_REGISTRY_SYNC_UPKEEP_QUEUE_SIZE" default:"10"`
 	LinkContractAddress                        string          `env:"LINK_CONTRACT_ADDRESS"`
 	LogFileDir                                 string          `env:"LOG_FILE_DIR"`
-	LogLevel                                   LogLevel        `env:"LOG_LEVEL"`
+	LogLevel                                   zapcore.Level   `env:"LOG_LEVEL" default:"info"`
```

Comment:
Interesting that we weren't setting the default before?

Reply:
There was custom logic in the config and elsewhere to parse and fallback to `DefaultLogLevel` instead.